.gitignore

```diff
@@ -39,4 +39,5 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
-/nbproject/private/
\ No newline at end of file
+/nbproject/private/
```

Comment:
The line `/nbproject/private` is not necessary.. the line you added below `/nbproject` will cover both cases (because it is the parent directory).